src/js/profile/indicator.js

```diff
@@ -38,7 +38,15 @@ export class Indicator extends Observable {
             }
         }
 
-        let c = new Chart(this.formattingConfig, this.subindicators, this.groups, detail, 'Percentage', indicator, title);
+
+        const configuration = detail.indicators[title].chartConfiguration;
+        const config = {
+            chart: detail.indicators[title].chartConfiguration,
+            formatting: this.formattingConfig
+        }
+        const indicators = detail.indicators;
+
+        let c = new Chart(config, this.subindicators, this.groups, indicators, 'Percentage', indicator, title);
```

Comment:
instead of creating the variable `indicators` in line 47, you could just use it here 
```js
let c = new Chart(config, this.subindicators, this.groups, detail.indicators, 'Percentage', indicator, title);
```

Reply:
detail needs to be refactored out of this class entirely